src/OpenLoco/Ui/ViewportInteraction.cpp

```diff
@@ -59,15 +59,15 @@ namespace OpenLoco::Ui::ViewportInteraction
     // 0x004CD974
     static bool _road(InteractionArg& interaction)
     {
-        auto* tileElement = reinterpret_cast<Map::tile_element*>(interaction.object);
+        auto* tileElement = reinterpret_cast<Map::TileElement*>(interaction.object);
         auto* road = tileElement->asRoad();
         if (road == nullptr)
             return false;
         if (!road->hasStationElement())
             return false;
 
-        Map::station_element* station = nullptr;
-        Map::tile tile{ interaction.x, interaction.y, tileElement };
+        Map::StationElement* station = nullptr;
+        Map::Tile tile{ { interaction.x / Map::tile_size, interaction.y / Map::tile_size }, tileElement };
```

Comment:
Linux CI is failing here. Can we do the resizing via constructors? Perhaps by using something like:
```suggestion
        Map::Tile tile{ Map::TilePos2(Map::Pos2(interaction.x, interaction.y)), tileElement };
```

Reply:
Yes that would be an idea. I was thinking i would bubble up the chain and make the interaction arg take a Pos2

Reply:
That would be even better. Currently it's just two coord_t's right?